content/applications/productivity/discuss.rst

```diff
@@ -1,105 +1,173 @@
 :show-content:
-:show-toc:
+:hide-page-toc:
 
 =======
 Discuss
 =======
 
-Odoo *Discuss* is an internal communication app that allows users to connect through messages,
-notes, and file sharing, either through a persistent chat window that works across applications, or
-through the dedicated *Discuss* dashboard.
+Odoo **Discuss** is an internal communication app that allows users to connect through messages,
+notes, file sharing, and video calls. **Discuss** enables communication through a persistent chat
+window that works across applications, or through the dedicated **Discuss** dashboard.
 
-.. _discuss_app/notification_preferences:
+Inbox, starred, and history
+===========================
 
-Choose notifications preference
-===============================
+Upon opening the :menuselection:`Discuss app`, the :guilabel:`Discuss` dashboard appears.
 
-Access user-specific preferences for the *Discuss* app by navigating to :menuselection:`Settings app
---> Users --> User --> Preferences tab`.
+On the :guilabel:`Discuss` dashboard, unread messages are visible in the :guilabel:`Inbox`.
+:guilabel:`Starred` is where starred messages are stored. :guilabel:`History` shows *Chatter*
+updates for records in the Odoo database the user has been assigned to, or tagged on.
```

Comment:
Suggestion to use icons for each of these section headings to match the UI.

"Chatter" does not need any formatting or capitalization
```suggestion
On the :guilabel:`Discuss` dashboard, unread messages are visible in the :icon:`fa-inbox`
:guilabel:`Inbox`. :icon:`fa-star-o` :guilabel:`Starred` is where starred messages are stored.
:icon:`fa-history` :guilabel:`History` shows chatter updates for records in the Odoo database the
user has been assigned to, or tagged on.
```